time start_time
time end_time
bool upper_bound
bool scale
---
string[] region_ids
float64[] estimates
